use-timescale/data-tiering/about-data-tiering.md

```diff
@@ -11,9 +11,10 @@ cloud_ui:
 
 # About data tiering
 
-Save on storage costs by tiering rarely used data to a low-cost object storage layer.
+Save on storage costs by tiering rarely used data to a low-cost object storage layer
+on Timescale Cloud.
 
-Timescale includes traditional disk storage, and a low-cost object storage
+Timescale Cloud service includes traditional disk storage, and a low-cost object storage
```

Comment:
Let's go with he following and I'll fix this on the follow-up with the name changes and the positioning

```suggestion
Timescale services include traditional disk storage, and a low-cost object storage
```

Reply:
@iroussos I highlighted Cloud to differentiate between MST and Timescale Cloud. Are we using soem other nomenclature now?

Reply:
Ok, let's leave this as is to differentiate from MST.
In general we do not use Timescale Cloud any more, just Timescale, but it makes sense in this case